bot/cogs/doc.py

```diff
@@ -391,7 +392,8 @@ async def get_command(self, ctx: commands.Context, symbol: commands.clean_conten
                     await error_message.delete(delay=NOT_FOUND_DELETE_DELAY)
                     await ctx.message.delete(delay=NOT_FOUND_DELETE_DELAY)
             else:
-                await ctx.send(embed=doc_embed)
+                doc_embed = await ctx.send(embed=doc_embed)
+                await wait_for_deletion(doc_embed, (ctx.author.id,), client=self.bot)
```

Comment:
No need to assign message to variable. Put send directly to call. Also no need to pass custom client.

Reply:
I don't agree with this, using a name makes this more readable compared to doing the whole await expression within an argument. Although an another name may be more fitting since `doc_embed` is already used to describe the embed object in the same scope.

Reply:
What names do you suggest? I kept it the same because it's still an embed... but not technically.

Reply:
I'd suggest `msg`.